src/maptm/src/maptm.h

```diff
@@ -40,6 +40,9 @@
 
 #define MAPTM_MODULE_NAME           "MAPTM"
```

Comment:
```suggestion
#define MAPT_MODULE_NAME            "MAPT"
```

1 - After cross-checking with the design for the northbound communication, it turns out that `module` should be `MAPT`, and not `MAPTM`. Which makes sense, because in future MAP-T could be managed by WANO rather than by a standalone MAPTM manager, so the parameters are really about MAPT, and not about MAPTM.

2 - Although `MAPTM_MODULE_NAME` was present already in the initial implementation, it wasn't used much until this PR. Therefore it makes sense to also adjust the name of the symbol now -- from `MAPTM_MODULE_NAME` to `MAPT_MODULE_NAME`. Suggestions for the four references included in the feedback.